packages/lesswrong/components/form-components/SelectLocalgroup.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import MenuItem from '@material-ui/core/MenuItem';
+import { useMulti } from '../../lib/crud/withMulti';
+
+/* A form input for selecting a localgroup -
+ * the options are a list of groups for which the current user is an organizer,
+ * or all groups if the user is an admin.
+ */
+const SelectLocalgroup = (props: any) => {
```

Comment:
Don't use `props: any`, give it a type.

Reply:
I'm not sure how to type this, though if you could suggest something I'd appreciate it.

This component gets called from `FormComponentInner.tsx` which passes in a bunch of props, and then inside my component I pass those through to `MuiTextField` which uses a bunch of those props. Should I manually type out all the props that are used by `MuiTextField` here, in addition to the one that I use in my component?

Reply:
You can make a type like the following:

```ts
type SelectLocalgroupProps = MuiTextFieldProps & { propWeUseFoo: string, propWeUseBar: string }
```

Reply:
I don't think I can do that since `MuiTextField` doesn't have a type for its props?